src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -92,6 +92,31 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 	func showTanScreen() {
 		let tanInputViewModel = TanInputViewModel(
 			coronaTestService: model.coronaTestService,
+			presentOverrideTestNotice: { [weak self] testType, feedbackHandler in
+				
+				let footerViewModel = FooterViewModel(
+					primaryButtonName: AppStrings.ExposureSubmission.OverwriteNotice.primaryButton,
+					isSecondaryButtonHidden: true
+				)
+
+				let overwriteNoticeViewController = TestOverwriteNoticeViewController(
+					testType: testType,
+					didTapPrimaryButton: {
+						self?.navigationController?.dismiss(animated: true) {
+							feedbackHandler(true)
+						}
+					},
+					didTapCloseButton: { [weak self] in
+						self?.navigationController?.dismiss(animated: true) {
+							feedbackHandler(false)
+						}
+					}
+				)
+
+				let footerViewController = FooterViewController(footerViewModel)
+				let topBottomViewController = TopBottomContainerViewController(topController: overwriteNoticeViewController, bottomController: footerViewController)
+				self?.push(topBottomViewController)
```

Comment:
I checked it out: it doesn't work, closes the flow when tapping on continue on the override notice screen. Thats why I suggested to extend the `CoronaTestQRCodeInformation` with the TAN case and use the same pattern as we do after scanning a QR code. It also shows the appropriate loading state on the confirmation screen as well as the error messages.

Reply:
That is exactly the expected behavior!

Reply:
The same thing that happened before: the positive test result is shown so the user can submit the keys